src/Table/Footer.jsx

```diff
@@ -120,10 +129,13 @@ Footer.propTypes = {
   onPageChange: PropTypes.func,
   numColumns: PropTypes.number.isRequired,
   numPages: PropTypes.number.isRequired,
+  showLastPage: PropTypes.bool,
+  isLoading: PropTypes.bool,
 };
 
 Footer.defaultProps = {
   onPageChange: () => {},
+  showLastPage: true,
```

Comment:
gotcha. For some reason I thought we decided in #frontend to keep the behavior consistent between the 2 tables